desktop.css

```diff
@@ -0,0 +1,521 @@
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 500;
+  margin-right: 12px;
+}
+
+.canopy {
+  display: flex;
+  align-items: baseline;
+}
+
+.common {
+  color: #6b778c;
+  font-size: 18px;
+  margin-left: 24px;
+  font-weight: 400;
+  margin-right: 12px;
+}
+
+h4 {
+  color: #344563;
+}
+
+.year {
+  display: flex;
+  list-style-type: disc;
+  color: #7a869a;
+}
+
+.year li {
+  list-style-type: disc;
+}
+
+ul.year {
+  list-style-type: circle;
+}
+
+ul.languages-btn {
+  margin: 8px 0;
+}
+
+li button {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  border: none;
+  border-radius: 8px;
+  padding: 4px 12px;
+  letter-spacing: 3%;
+}
+
+.languages-btn {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  list-style: none;
+  height: 24px;
+  margin-top: 0;
+  margin-bottom: 24px;
+}
+
+.btn {
+  padding: 4px;
+  font-size: 12px;
+}
+
+.big-btn {
+  width: 124px;
+  height: 48px;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  text-align: center;
+  margin: 26px 0;
+  background-color: rgba(255, 255, 255, 0);
+}
+
+.big-btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.big-btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.big-btn:disabled {
+  background-color: white;
+  color: #5e6c84;
+}
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 48%;
+  padding: 30px;
+}
+
+.photoshoot-D-1 {
+  border-radius: 8px;
+  padding: 20px;
+  width: 48%;
+}
+
+.photoshoot-3 {
+  display: none;
+}
+
+.card1 {
+  grid-area: first;
+}
+
+.card2 {
+  grid-area: second;
+}
+
+.card3 {
+  grid-area: third;
+}
+
+.card4 {
+  grid-area: fourth;
+}
+
+.photoshoot-1 {
+  display: none;
+}
+
+h5 {
+  font-weight: 700;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  width: 327px;
+  height: 101px;
+  padding-top: 114px;
+  margin-bottom: 12px;
+}
+
+.about-me {
+  background-color: #fff;
+  border-top-right-radius: 118px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.about-p {
+  color: #344563;
+  font-family: Poppins, 'Times New Roman', Times, serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 24px;
+  height: 146px;
+  margin-top: 12px;
+}
+
+.back-end {
+  list-style-type: disc;
+}
+
+.social-icons {
+  margin: 20px 0;
+}
+
+.about-btn {
+  width: auto;
+}
+
+.lang {
+  font-size: 20px;
+  font-weight: 500;
+  list-style: none;
+}
+
+.myself {
+  padding-left: 12px;
+  width: 40%;
+}
+
+.lang-png {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 36px;
+  padding-top: 114px;
+}
+
+.line {
+  border-bottom: solid 1px #dfe1e6;
+  padding: 23px 36px;
+}
+
+.about-div {
+  padding-left: 36px;
+}
```

Comment:
- [ ] You have done an excellent job with your desktop version design but I would suggest adding a % for the width in this section because according to the Figma design the left part should take around 40% and the right side around 60% but your design shows a bigger left side. Please, fix this width issue to match the template :wink: :raised_hands: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144673915-f05e4995-aef7-4414-b72f-f55662911a2c.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144673951-ddd6616e-7d68-40dc-9d5b-c8fce3929e88.png)